package/MDAnalysis/core/universe.py

```diff
@@ -236,39 +236,56 @@ def universe(self):
         # It is also cleaner than a weakref.
         return self
 
-    def load_new(self, filename, **kwargs):
-        """Load coordinates from *filename*, using the suffix to detect file format.
-
-        :Arguments:
-             *filename*
-                 the coordinate file (single frame or trajectory) *or* a list of
-                 filenames, which are read one after another.
-             *format*
-                 provide the file format of the coordinate or trajectory file;
-                 ``None`` guesses it from the file extension. Note that this
-                 keyword has no effect if a list of file names is supplied because
-                 the "chained" reader has to guess the file format for each
-                 individual list member [``None``]
-                 Can also pass a subclass of :class:`MDAnalysis.coordinates.base.Reader`
-                 to define a custom reader to be used on the trajectory file.
-             *kwargs*
-                 Other kwargs are passed to the trajectory reader (only for advanced use)
-
-        :Returns: (filename, trajectory_format) or ``None`` if *filename* == ``None``
-        :Raises: :exc:`TypeError` if trajectory format can not be
+    def load_new(self, filename, format=None, in_memory=False, **kwargs):
+        """Load coordinates from *filename*, using the suffix to detect file
+        format.
+
+        Parameters
+        ----------
+        filename : str
+            the coordinate file (single frame or trajectory) *or* a list of
+            filenames, which are read one after another.
+        format : str (optional)
+            provide the file format of the coordinate or trajectory file;
+            ``None`` guesses it from the file extension. Note that this
+            keyword has no effect if a list of file names is supplied because
+            the "chained" reader has to guess the file format for each
+            individual list member [``None``] Can also pass a subclass of
+            :class:`MDAnalysis.coordinates.base.Reader` to define a custom
+            reader to be used on the trajectory file.
+        in_memory : bool (optional)
+            Directly load trajectory into memory
```

Comment:
Not really right now. They are given to the `transfer_to_memory` function with different names. My current plan is to get the other PR's merged and then refactor `transfer_to_memory` before I take a long hard look at this and write a lot of test cases  to catch all possible ways to call this.